packages/buidler-core/src/internal/buidler-evm/stack-traces/solidityTracer.ts

```diff
@@ -728,7 +751,8 @@ export class SolidityTracer {
 
     return (
       calledFunction === undefined &&
-      trace.bytecode.contract.fallback === undefined
+      trace.bytecode.contract.fallback === undefined &&
+      trace.bytecode.contract.receive === undefined
```

Comment:
I don't get why the receive function should be included here.

Reply:
I added this so that the `0_6/call-message/no-calls/revert-without-message-in-receive` passes. But this change makes `0_6/fallback-and-receive/with-calldata-with-value/no-fallback-and-receive` and `0_6/fallback-and-receive/with-calldata-without-value/no-fallback-and-receive` fail.